mcp-tools/maos-mcp-hub/lib/confluence/client.py

```
@@ -0,0 +1,231 @@
+import os
+from typing import Optional
+
+from aiolimiter import AsyncLimiter
+
+from lib.common.http import request_json
+
+
+class ConfluenceClient:
+    """Async HTTP client for Confluence Cloud REST API v2."""
+
+    def __init__(self, cloud_id: Optional[str] = None):
```

Ruff (0.15.9)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)